<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kindle 下载助手</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/resource/kindle.png</normaloff>:/icon/resource/kindle.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QGroupBox" name="listBox">
       <property name="title">
        <string>下载列表</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item alignment="Qt::AlignRight">
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="radioEBOK">
             <property name="text">
              <string>电子书</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioPDO">
             <property name="text">
              <string>个人文档</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fetchButton">
             <property name="text">
              <string>获取下载列表</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="bookView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>目标文件夹</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outDirEdit">
            <property name="text">
             <string>DOWNLOADS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="text">
             <string>浏览...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="text">
             <string>下载全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dedrmCkb">
            <property name="text">
             <string>DeDRM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>输出</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTextBrowser" name="logBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft YaHei UI'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QGroupBox" name="settingsBox">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="loginGroupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="radioCOM">
               <property name="text">
                <string>美亚(.com)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioCN">
               <property name="text">
                <string>中亚(.cn)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioJP">
               <property name="text">
                <string>日亚(.jp)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioDE">
               <property name="text">
                <string>德亚(.de)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="loginButton">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="cookiesGroupBox">
          <property name="title">
           <string>Cookies</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="cookieTextEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioFromInput">
               <property name="text">
                <string>来自输入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioFromBrowser">
               <property name="text">
                <string>来自浏览器</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>CSRF Token</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="csrfEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>文件名截断</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="cutLengthSpin">
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QGroupBox" name="copyrightBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>隐私声明：我们不会收集任何用户信息，请放心使用</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Copyright 2022 © [yihong0618](https://github.com/yihong0618) and [frostming](https://github.com/frostming)</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>GitHub: &lt;https://github.com/yihong0618/Kindle_download_helper&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::MarkdownText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>License: GPL V3</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
